apps/admin-ui/src/component/reservations/requested/hooks/queries.ts

```diff
@@ -70,6 +70,7 @@ const SPECIALISED_SINGLE_RESERVATION_FRAGMENT = gql`
   }
 `;
 
+// TODO replace with relay query
```

Comment:
Check that this still exists (in the API), i.e. the query works.

Reply:
Fine. Still works. `reservationUnitByPk` got removed, the `reservationByPk` is removed in the next PR.